openpype/tools/sceneinventory/view.py

```diff
@@ -5,13 +5,14 @@
 from Qt import QtWidgets, QtCore
 import qtawesome
 
-from avalon import io, api
+from avalon import io
 
 from openpype import style
 from openpype.pipeline import (
     HeroVersionType,
     update_container,
     remove_container,
+    discover_inventory_actions,
```

Comment:
Command at the end is redundant - it should work as is but might be cleaner without?
```suggestion
    discover_inventory_actions
```